cfme/common/provider_views.py

```diff
@@ -187,7 +187,7 @@ class ProvidersManagePoliciesView(BaseLoggedInPage):
      Provider's Manage Policies view
     """
     policies = VersionPick({Version.lowest(): DynaTree('protect_treebox'),
-                            '5.7': BootstrapTreeview('protectbox')})
```

Comment:
**REQUIRED**
`VersionPick` is not needed here.

Reply:
Just for see that i understand , you mean that older version then 5.7 is no needed so we don't need versionPick , so i will change it to :+1: 
policies = CheckableBootstrapTreeview('protectbox')

Reply:
That's correct